web.config

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Is this file necessary?

Reply:
Confirmed with Fan: This file is necessary when deployed to azure app service to tell IIS that this project is a nodejs project.